src/CitizenProject/CitizenProjectAuthority.php

```diff
@@ -41,6 +41,16 @@ public function changePrivilege(Adherent $adherent, CitizenProject $citizenProje
         if (!$membership = $this->getCitizenProjectMembership($adherent, $citizenProject)) {
             return;
         }
+        if (CitizenProjectMembership::CITIZEN_PROJECT_ADMINISTRATOR === $privilege) {
+            $membershipAdmins = $this->membershipRepository->findPriviledgedMemberships(
+                $citizenProject->getUuid(),
+                [CitizenProjectMembership::CITIZEN_PROJECT_ADMINISTRATOR]
+            );
+            if (1 === \count($membershipAdmins)) {
```

Comment:
Okay i think the functional behavior here is weird.

If more than 1 administrator is found: there won't be any privilege changed and nobody will be notified.
If count is different than 1, an Exception should be thrown to tell the user what's happening?
Or maybe a flash message should be displayed ?

Reply:
I will change it to put all admin as followers